CUETools.Codecs.FlaCuda/FlaCudaWriter.cs

```diff
@@ -1972,70 +1972,70 @@ struct FlakeEncodeParams
 		public int compression;
 
 		// stereo decorrelation method
-		// set by user prior to calling flake_encode_init
+		// set by user prior to calling Flake_encode_init
 		// if set to less than 0, it is chosen based on compression.
 		// valid values are 0 to 2
 		// 0 = independent L+R channels
 		// 1 = mid-side encoding
 		public bool do_midside;
 
 		// block size in samples
-		// set by the user prior to calling flake_encode_init
+		// set by the user prior to calling Flake_encode_init
 		// if set to 0, a block size is chosen based on block_time_ms
 		// can also be changed by user before encoding a frame
 		public int block_size;
 
 		// block time in milliseconds
-		// set by the user prior to calling flake_encode_init
+		// set by the user prior to calling Flake_encode_init
 		// used to calculate block_size based on sample rate
 		// can also be changed by user before encoding a frame
 		public int block_time_ms;
 
 		// padding size in bytes
-		// set by the user prior to calling flake_encode_init
+		// set by the user prior to calling Flake_encode_init
 		// if set to less than 0, defaults to 4096
 		public long padding_size;
 
 		// minimum LPC order
-		// set by user prior to calling flake_encode_init
+		// set by user prior to calling Flake_encode_init
 		// if set to less than 0, it is chosen based on compression.
 		// valid values are 1 to 32
 		public int min_prediction_order;
 
 		// maximum LPC order
-		// set by user prior to calling flake_encode_init
+		// set by user prior to calling Flake_encode_init
 		// if set to less than 0, it is chosen based on compression.
 		// valid values are 1 to 32 
 		public int max_prediction_order;
 
 		public int orders_per_window;
 
 		// minimum fixed prediction order
-		// set by user prior to calling flake_encode_init
+		// set by user prior to calling Flake_encode_init
 		// if set to less than 0, it is chosen based on compression.
 		// valid values are 0 to 4
 		public int min_fixed_order;
 
 		// maximum fixed prediction order
-		// set by user prior to calling flake_encode_init
+		// set by user prior to calling Flake_encode_init

```

Comment:
Please undo. The lower case letter at the beginning of the method name should stay, like in other methods.